pandas/io/tests/test_clipboard.py

```diff
@@ -71,6 +71,7 @@ def check_round_trip_frame(self, data_type, excel=None, sep=None):
     def test_round_trip_frame_sep(self):
```

Comment:
@gfyoung not sure what you exactly mean by 'round-trip tests', I added a test for pipe-delimiter though.